### ISSUE TITLE
How to train my own dataset?

### ISSUE THREAD

**Issue by CRonaldo1997** [qfgaohao/pytorch-ssd#2]
Hi thanks for your work, how can I train the model using own dataset？ Thanks

**Comment by qfgaohao** [comment]
Hi @CRonaldo1997 , I think the easiest way is as follows:
1. Convert you dataset into Pascal VOC format.
2. run "python train_ssd.py --datasets <you  train dataset>  --validation_dataset <you validation dataset> --net mobilenet-v1-ssd --base_net models/mobilenet_v1_with_relu_69_5.pth  --batch_size 24 --num_epochs 200 --scheduler cosine --lr 0.01 --t_max 200"

Feel free to change the parameters.

**Comment by qfgaohao** [comment]
It's ok now. Plz refer to Issue #4

**Comment by kurianbenoy** [comment]
> Hi @CRonaldo1997 , I think the easiest way is as follows:
> 
>     1. Convert you datasets into Pascal VOC format.


For a dataset without Pascal VOOC format, removing years from data loader will work out right @qfgaohao ?
